src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables.scss";
```

Comment:
And here, also very tight, haha. If you see, image overlaps text. I think you can use scale with less value, maybe 1.05. Try this, if you want

![image](https://github.com/mate-academy/layout_landing-page/assets/142230575/9175bbf3-39a1-4320-a9dd-c45f80a27732)